src/components/CalendarDay.jsx

```diff
@@ -48,6 +55,9 @@ export default class CalendarDay extends React.Component {
         onClick={e => this.onDayClick(day, e)}
       >
         <span className="CalendarDay__day">{day.format('D')}</span>
+        {details && (
+          <span className="CalendarDay__details">{details}</span>
```

Comment:
Indeed. In re:handing off render entirely, the reason I didn't go that way is because if the user still wants the default formatting for the number, they would have to know to wrap in `.CalendarDay__day`; I didn't think the API should expose implementation.

Could you say more about `React.Children.only`? The documentation for that method doesn't really tell me _why_ I would use it. 😕